KEP-4009: Adding a proposal section (#4487)
kubernetes/enhancements: keps/sig-node/4009-add-cdi-devices-to-device-plugin-api/README.md

* Allow Device Plugin authors to forward device requests to CRI runtimes as a CRI field.
* Allow Device Plugin authors to use CDI to define the modifications required for containerised environments.

## Proposal

> The adds a mechansims for device plugin authors to specify devices using their Container Device Interface (CDI) names. The names of the requested devices are passed down as CRI fields to CRI runtimes which are ultimately responsible for making the requested devices accessible from a container.

Reviewer: ```suggestion
We propose a mechanism for device plugin authors to specify devices using Container Device Interface (CDI) names. The names of the requested devices are passed down as CRI fields to CRI runtimes which are ultimately responsible for making the requested devices accessible from a container.
```
